### ISSUE TITLE
Wrong code published to npm

### ISSUE THREAD

**Issue by aliismael** [hapijs/hapi-auth-cookie#180]
Dear Colleagues,

When I run  
```javascript
npm install hapi-auth-cookie@7.0.0

```
I get code that does not look like the one on the master branch on github!  I get something like this in the lib/index.js file:

```javascript
'use strict';

// Load modules

const Boom = require('boom');
const Hoek = require('hoek');
const Joi = require('joi');

// Declare internals

const internals = {};


exports.register = function (server, options, next) {

    server.auth.scheme('cookie', internals.implementation);
    next();
};


exports.register.attributes = {
    pkg: require('../package.json')
};

.....
....
....

```

Did someone push the wrong code base to npm?

Thanks in advance for your help.

Yours truly,
Ali

**Comment by mrlannigan** [comment]
@aliismael The code for version [7.0.0](https://github.com/hapijs/hapi-auth-cookie/tree/22dfeb105705986728d5e97f4183f4d460c72608) is what was published to npm@7.0.0. I had merged a PR to upgrade this module to support hapi v17 yesterday and just a few minutes ago, cut the release to npm for 8.0.0. My apologies for the confusion.

**Comment by aliismael** [comment]
Dear Julian,

You are so kind, no need for apologies.

I am upgrading my OpenID-Connect plugin for hapi 17 as well, and I feel
what you are going through.

Thank you so much, I will keep you updated if I encounter any issue with
version 8.0.0 of your plugin.

Yours truly,,
Ali




Yours truly,

Ali Ismael, Ph.D.
Gaaiat, Inc
